smaug/python/node.py

```diff
@@ -0,0 +1,126 @@
+import sys
```

Comment:
It's worth a doc comment at the top explaining what Node is.

Reply:
Added a doc comment.